adapters/repos/db/ranked_fusion_test.go

```diff
@@ -364,52 +404,6 @@ func TestRFJourney(t *testing.T) {
 		require.Equal(t, float32(0.016393442), results[1].Score)
 	})
 
-	// Check basic search with one property
-	results_set_1, err := repo.VectorSearch(context.TODO(), dto.GetParams{
```

Comment:
**question (non-blocking):** is it intended to be deleted ?

Reply:
yes, this is the failing test.  I replaced it with the smaller test above